src/Compiler/Compiler.cs

```diff
@@ -86,6 +86,22 @@ public async ValueTask<CompiledAssembly> CompileAsync(
         return result.CompiledAssembly;
     }
 
+    public string FormatCode(string code)
+    {
+        try
+        {
+            var syntaxTree = CSharpSyntaxTree.ParseText(code);
```

Comment:
I wonder whether we need to pass parse options for best results; but that can definitely wait for a follow up.